### PR TITLE
test for polyfill combination issues

### PR DESCRIPTION
**Pull request by romainmenke**
_Fell down a rabbit hole 🙂_ 

I was initially lookup for a way to run tests for each polyfill in the first version that does not require it.
If config states `edge <13` then I wanted to run tests in edge 13 to ensure that the polyfill is not needed.

Messing around with the test server I at one point loaded all polyfills needed for a browser instead of only the polyfills for the feature.

```diff
test/polyfills/test-runner.handlebars
- <script src="/polyfill.js?includePolyfills={{{includePolyfills}}}&always={{{always}}}&feature={{features}}"></script>
+ <script src="/polyfill.js?includePolyfills={{{includePolyfills}}}&always={{{always}}}"></script>
```

A lot of tests started failing because of this.

Main issues :
- some polyfills breaks others
- some tests have a feature test which never passes as the polyfill isn't loaded

I got IE11 passing again after refactoring Symbol and some smaller issues.
The quality of the code here is lacking, was mainly focussed on documenting the issue.

Might be best to gather a list of detailed issues so we can resolve them.

@JakeChampion thoughts?

### CODE REVIEW

File: polyfills/Symbol/match/config.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@ spec = "http://www.ecma-international.org/ecma-262/6.0/index.html#sec-symbol.mat
 docs = "https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Symbol"
 
 [browsers]
-edge = "<14"
+edge = "<79"
```
Comment (JakeChampion) [inline polyfills/Symbol/match/config.toml:8]: 👍

**Comment by JakeChampion** [comment]
@romainmenke, good idea to run all the tests for all polyfills targeted to the browser in one test run rather than in separate/isolated iframes to ensure the polyfills work together 👍

**Comment by romainmenke** [comment]
@JakeChampion Can you setup a branch with just this change :

```diff
test/polyfills/test-runner.handlebars
- <script src="/polyfill.js?includePolyfills={{{includePolyfills}}}&always={{{always}}}&feature={{features}}"></script>
+ <script src="/polyfill.js?includePolyfills={{{includePolyfills}}}&always={{{always}}}"></script>
```

Then we can open PR's against that branch to keep reviews small and merge that one when all tests are green again.

**Comment by JakeChampion** [comment]
> @JakeChampion Can you setup a branch with just this change :
> 
> ```diff
> test/polyfills/test-runner.handlebars
> - <script src="/polyfill.js?includePolyfills={{{includePolyfills}}}&always={{{always}}}&feature={{features}}"></script>
> + <script src="/polyfill.js?includePolyfills={{{includePolyfills}}}&always={{{always}}}"></script>
> ```
> 
> Then we can open PR's against that branch to keep reviews small and merge that one when all tests are green again.

Done -- https://github.com/Financial-Times/polyfill-library/pull/924

**Comment by JakeChampion** [comment]
Should this be closed in favour of #924 @romainmenke ?

**Comment by romainmenke** [comment]
Yes! I'll close it.

**Pull request closed without merging**